content/doc/pipeline/index.adoc

```diff
@@ -122,27 +157,43 @@ Pipeline: Step API
 
 
 === More Information
-As with any Jenkins plugin, you have the option of installing the Pipeline plugin from the link:https://wiki.jenkins-ci.org/display/JENKINS/Plugins[Plugins web page], but using the Plugin Manager interface is preferred because you do not then have to make allowances for plugin dependencies or compatibility issues.
-To investigate Pipeline without installing Jenkins separately or accessing your production system, you can run a link:https://github.com/jenkinsci/workflow-plugin/blob/master/demo/README.md[Docker demo] of Pipeline functionality.
-
-Pipeline-related plugins are regularly "whitelisted" as compatible with or designed for Pipeline usage. For more information, see the link:https://github.com/jenkinsci/workflow-plugin/blob/541faf611659e1e6b8f2cbbd3435756b27633db4/COMPATIBILITY.md[Plugin Compatibility With Pipeline] wiki.
 
-Note: Several plugins available in the Jenkins ecosystem but not actually related to the Pipeline feature set described in this guide also use the terms "pipeline," "DSL," and "Job DSL" in their names. For example:
+As with any Jenkins plugin, you have the option of installing the Pipeline
+plugin from the link:https://wiki.jenkins-ci.org/display/JENKINS/Plugins[Plugins
+web page], but using the Plugin Manager interface is preferred because you do
+not then have to make allowances for plugin dependencies or compatibility
+issues.  To investigate Pipeline without installing Jenkins separately or
+accessing your production system, you can run a
+link:https://github.com/jenkinsci/workflow-plugin/blob/master/demo/README.md[Docker
+demo] of Pipeline functionality.
```

Comment:
This link was broken by the pipeline repo split.

It seems to be here: https://github.com/jenkinsci/workflow-aggregator-plugin/blob/master/demo/README.md